src/clr-angular/layout/tabs/tabs.ts

```diff
@@ -51,13 +54,28 @@ import { ClrCommonStrings } from '../../utils/i18n/common-strings.interface';
         </ng-container>
     `,
   providers: [IfActiveService, IfOpenService, TabsService, TABS_ID_PROVIDER],
+  host: {
+    '[class.tabs-vertical]': 'tabsService.orientation === "vertical"',
+  },
 })
 export class ClrTabs implements AfterContentInit {
-  @ContentChildren(ClrTabLink, { descendants: true })
-  tabLinkDirectives: QueryList<ClrTabLink>;
+  @Input('orientation')
```

Comment:
I'm wondering if we want to call this `layout` instead? Orientation to me is like portrait/landscape.

Reply:
In addition to @gnomeontherun's comment this input needs to be prefixed with `clr` per Clarity naming conventions: https://github.com/vmware/clarity/blob/master/CODING_GUIDELINES.md#naming-conventions

Reply:
Both comments addressed. Using clrTabsLayout now.